main-command/src/main/java/sbt/internal/classpath/WrappedLoader.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * sbt
+ * Copyright 2011 - 2018, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * Licensed under Apache License 2.0 (see LICENSE)
+ */
+
+package sbt.internal.classpath;
+
+import java.net.URL;
+import java.net.URLClassLoader;
+
+public class WrappedLoader extends URLClassLoader {
```

Comment:
great.
Could you link to https://docs.oracle.com/javase/8/docs/api/java/lang/ClassLoader.html in the commit message as the explanation of "parallel capable class loaders"?

Reply:
Sure thing.

Reply:
I improved the commit message in https://github.com/sbt/sbt/pull/4769/commits/625470cdd5e219ff0e1436c6d157c03e41c935cb